lib/coverage-middleware.js

```diff
@@ -18,23 +19,24 @@ function reporterForName(reporterName) {
   return require(path.join(__dirname, 'reporters', reporterName + '-reporter'));
 }
 
-// TODO handle multiple reporters
-function loadReporter(blanketOptions) {
-  var reporterName = blanketOptions.cliOptions.reporters[0] || 'null';
-
-  var OutputReporter = reporterForName(reporterName);
+function loadReporter(blanketOptions, name) {
+  var OutputReporter = reporterForName(name);
   var reporter = new OutputReporter(blanketOptions);
   return reporter;
 }
 
 module.exports = function(options) {
 
-
   var blanketOptions = loadBlanketOptions(options);
-  var reporter = loadReporter(blanketOptions);
+  var reporters = _.map(blanketOptions.cliOptions.reporters, function (reporter) {
```

Comment:
I usually prefer the native `.forEach` and `.map` as well. I was a bit lazy and I didn't check what versions of node you were targeting, so I figured it was safer to use `loadash` since you already had a dependency on it. I just checked and `0.10.x` appears to have full support for ES5, so I can make another PR changing that, no problem.